### ISSUE TITLE
alex button iffy

### ISSUE THREAD

**Issue by elijahparker000** [elijahparker000/JeopardyPiV2#2]
alex's button doesn't always show the indicator bars when released. Not sure why. Could just be a sucky button

**Comment by elijahparker000** [comment]
Wait actually I'm pretty sure this is solely because I've been releasing it too early. It'll still debounce even if the clue is already shown

**Comment by elijahparker000** [comment]
Yeah something funky's going on with alex's buzzer, but it's not the button itself. I think it has to do with the indicator textboxes. Maybe switching those to widgets would help?

**Comment by elijahparker000** [comment]
I believe this has been resolved, but I'll leave the issue open for a little while. The problem seems to have been related to hardware. Long wires and low voltages mean incorrect rising and falling edges. Some of the falling edges were being interpreted as rising edges. I've put a 0.1uF cap across alex's GPIO pin and GND which apparently helps. I found the recommendation here: https://forums.raspberrypi.com/viewtopic.php?t=82723      I just clicked through 2 full games with no issues. Also, I assume these caps should be added to all buzzers to prevent any errors with players ringing in. PCB will need to be modified so it's a good thing I didn't order it today lol
